lib/spack/spack/cmd/style.py

```diff
@@ -38,7 +38,7 @@ def grouper(iterable, n, fillvalue=None):
 #: List of directories to exclude from checks.
 exclude_directories = [spack.paths.external_path]
 
-#: max line length we're enforcing (note: this duplicates what's in .flake8)
+# max line length we're enforcing (note: this duplicates what's in .flake8)
```

Comment:
`sphinx.ext.autodoc` didn't like this for some reason, couldn't figure out why so I just removed it from the docs.